cmd/registry/conformance/conformance-task.go

```diff
@@ -107,40 +110,77 @@ func (task *ComputeConformanceTask) Run(ctx context.Context) error {
 		return err
 	}
 
-	// Generate conformance report
+	// Run the linters and compute conformance report
 	conformanceReport := initializeConformanceReport(task.Spec.GetName(), task.StyleguideId)
-	err = task.computeConformanceReport(ctx, root, conformanceReport)
-	if err != nil {
-		log.Errorf(ctx,
-			"Failed to compute the conformance for spec %s: %s",
-			task.Spec.GetName(),
-			err,
-		)
-		return err
+	guidelineReportsMap := make(map[string]int)
+	for _, metadata := range task.LintersMetadata {
```

Comment:
Moved the for loop out of `computeConformanceReport` out here. This was done for better testability. We can now pass different combinations of `linterResponse` and `metadata` to `computeConformanceReport` and have unit test coverage.